src/main.c

```diff
@@ -255,7 +274,11 @@ int main(int argc, char** argv)
 
   /* create our main classes on the stack*/
   struct imv_loader ldr;
-  imv_init_loader(&ldr);
+  if(g_options.binary_stdin){
```

Comment:
If we pre-set `stdin_buffer` to NULL and `stdin_buffer_size` to 0 when we declare them, we don't need this if.